src/main/java/com/work/management/service/employee/impl/EmployeeServiceImpl.java

```diff
@@ -95,7 +96,12 @@ public EmployeeDto updateEmployeeEntity(EmployeeDto employeeDto) {
 
   @Override
   @Transactional(isolation = Isolation.SERIALIZABLE, propagation = Propagation.REQUIRED)
-  public List<Employee> bulkUpdate(BulkEmployeeDto bulkEmployeeDto) {
+  public ImmutableList<Employee> bulkUpdate(BulkEmployeeDto bulkEmployeeDto) {
+    if (!(bulkEmployeeDto.getEmployeeIds().size() == ImmutableSet
+        .copyOf(bulkEmployeeDto.getEmployeeIds()).size())) {
```

Comment:
Why do you want to copy to set just for checking the size. Remove Immutable.copyOf...just getEmployeeIds().size()